### ISSUE TITLE
Image widget "Insert From URL" should be optional

### ISSUE THREAD

**Issue by madsem** [netlify/netlify-cms#5532]
The feature introduced here: https://github.com/netlify/netlify-cms/pull/4700/files
should be optional imho.

It's a great feature, but in some cases it encourages hotlinking haha.
I would prefer to be able to turn it off, just maybe like
```js
label: "Hero Image",
      name: "image",
      widget: "image",
      choose_url: false,
      allow_multiple: false,
      default: "/img/shareable-default.jpg",
      i18n: true,
```

**Comment by erezrokah** [comment]
Hi @madsem, great suggestion. It would make for a great first contribution.

The relevant code is here:
https://github.com/netlify/netlify-cms/blob/c353eb761e8cca6638f95f7a75473d0f641196c0/packages/netlify-cms-widget-file/src/withFileControl.js#L303
https://github.com/netlify/netlify-cms/blob/c353eb761e8cca6638f95f7a75473d0f641196c0/packages/netlify-cms-widget-file/src/withFileControl.js#L321

in case someone would like to pick this up.

**Comment by madsem** [comment]
Thanks for pointing me in the right direction, I will have a looksy :)

**Comment by Edald123** [comment]
Hi! I would like to try this issue if no one is working on it yet. Is it possible?
If yes, is there more information I would need to start working on the issue? Thank you!

**Comment by madsem** [comment]
I have a huge work load already, so i would gladly give you the reigns :)

**Comment by Edald123** [comment]
Great, I can take it and start working on it. :D

**Comment by Edald123** [comment]
Hi! I'm working on the issue and I would like to ask:
- What would be the expected behavior of this feature?
- What would be the way that you expect to activate and de-activate the widget?
Thank you!

**Comment by madsem** [comment]
Imho it should be activated by default, just as it is now, that way it's backwards compatible.

Accept an option to deactivate the url link, something like `from_url: false`.

And I personally would expect the behavior to be: Once you set from_url to false, the link to add from a url isn't shown anymore in the UI. So that a content manager can only upload images to media library or select an existing image.

**Comment by Edald123** [comment]
Thank you! I'm going to be working on that!

**Comment by Edald123** [comment]
PR created. Let me know if any changes are needed, thank you!

**Comment by erezrokah** [comment]
Closed via https://github.com/netlify/netlify-cms/pull/5572

**Comment by madsem** [comment]
@erezrokah when will the next version be released that includes this feature?

**Comment by erezrokah** [comment]
@madsem, this was released in [netlify-cms@2.10.141](https://github.com/netlify/netlify-cms/releases/tag/netlify-cms%402.10.141)  and [netlify-cms-app@2.15.21](https://github.com/netlify/netlify-cms/releases/tag/netlify-cms-app%402.15.20)
